modules/local/multiqc_custom_phantompeakqualtools.nf

```diff
@@ -1,21 +1,9 @@
-// Import generic module functions
-include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
-
-params.options = [:]
-options        = initOptions(params.options)
-
 process MULTIQC_CUSTOM_PHANTOMPEAKQUALTOOLS {
-    publishDir "${params.outdir}",
-        mode: params.publish_dir_mode,
-        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:[:], publish_by_meta:[]) }
-
     //TODO substitute with a newest tag (see https://github.com/BioContainers/containers/issues/416)
```

Comment:
Remove/address this comment?

Reply:
Addressed here 50c0410850b88b2278128d65118a5c283022092b